sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -517,9 +517,9 @@ class CustomFormModel(object):
         Status indicating the model's readiness for use,
         :class:`~azure.ai.formrecognizer.CustomFormModelStatus`.
         Possible values include: 'creating', 'ready', 'invalid'.
-    :ivar ~datetime.datetime requested_on:
+    :ivar ~datetime.datetime training_started_on:
         The date and time (UTC) when model training was requested.
```

Comment:
should we change it here for `training was started?`

Reply:
Good catch, will update in both spots